build/dependencies.props

```diff
@@ -8,6 +8,7 @@
     <MicrosoftAspNetCoreHttpConnectionsClientPackage3_0Version>3.0.0</MicrosoftAspNetCoreHttpConnectionsClientPackage3_0Version>
     <MicrosoftAspNetCoreHttpConnectionsClientPackage3_1Version>3.1.9</MicrosoftAspNetCoreHttpConnectionsClientPackage3_1Version>
     <MicrosoftAspNetCoreHttpConnectionsClientPackage5_0Version>5.0.1</MicrosoftAspNetCoreHttpConnectionsClientPackage5_0Version>
+    <MicrosoftAspNetCoreHttpConnectionsClientPackage7_0Version>7.0.0-preview.7.22376.6</MicrosoftAspNetCoreHttpConnectionsClientPackage7_0Version>
```

Comment:
Althouth .NET SDK latest version is 7.0.100-preview.7.22377.5
the latest version of `Microsoft.AspNetCore.Http.Connections.Client` is 7.0.0-preview.7.22376.6 (refer to [Nuget.org](https://www.nuget.org/packages/Microsoft.AspNetCore.Http.Connections.Client/#versions-body-tab))

Reply:
Not a problem. We'll upgrade to net 7.0 release version before our final release. Make it up to date to easier find potential breaks in an early stage.